app/experimenter/experiments/api/v5/mutations.py

```diff
@@ -68,8 +69,28 @@ def mutate(cls, root, info, input: ExperimentInput):
         return handle_with_serializer(cls, serializer)
 
 
+class EndExperiment(graphene.Mutation):
+    nimbus_experiment_id = graphene.Int()
+    message = ObjectField()
+    status = graphene.Int()
+
+    class Arguments:
+        input = ExperimentIdInput(required=True)
+
+    @classmethod
+    def mutate(cls, root, info, input: ExperimentIdInput):
+        exp = NimbusExperiment.objects.get(id=input.id)
+        kinto.tasks.nimbus_end_experiment_in_kinto.delay(exp.id)
```

Comment:
Based on [feedback in the other pr](https://github.com/mozilla/experimenter/pull/4541#discussion_r572259641) we might want to set a db flag here instead of invoking the task